crates/bevy_gltf/src/loader.rs

```diff
@@ -368,6 +368,12 @@ async fn load_gltf<'a, 'b>(
                         scale: bevy_math::Vec3::from(scale),
                     },
                 },
+                extras: node
+                    .mesh()
+                    .and_then(|mesh| mesh.extras().as_ref())
+                    .map(|extras| super::GltfExtras {
+                        value: extras.get().to_string(),
+                    }),
```

Comment:
Those are not the node extras, those are the node' mesh extras

Maybe add a `extras` field with the node extras, and a `mesh_extras` field with those from the mesh?

Reply:
Yes. Because nodes, meshes, primitives and materials can have extra properties, I added the appropriate fields to the Gltf structures